src/options/proof_options.toml

```diff
@@ -159,6 +159,14 @@ name   = "Proof"
   default    = "false"
   help       = "Prune unused input assumptions from final scope"
 
+[[option]]
+  name       = "proofDefineSkolems"
+  category   = "expert"
+  long       = "proof-define-skolems"
```

Comment:
Rename this to `alethe-proof-define-skolems`.

Reply:
done

Reply:
I misread Andy's comment, lol. But I prefer `proof-alethe...` since this is how other similar options are named. Thoughts, @ajreynol @hansjoergschurr ?